cmd/create.go

```diff
@@ -291,6 +302,15 @@ func (cc *createCmd) createDeployment() error {
 		}
 	}
 
+	for _, flagVar := range cc.flagVariables {
```

Comment:
when a variable is passed using a variable flag, draft shouldn't prompt for it interactively, so you may need to use  RunPromptsFromConfigWithSkips and extract the variables from the flag prior to the RunPromptsFromConfig() call

Reply:
got it, making the changes